### ISSUE TITLE
[2020-07-14][이예은] Image Tagging을 위한 이미지 데이터 기본 전처리 및 모델링

### ISSUE THREAD

**Issue by Yeeun-Lee** [boogle-lab/ssho-daily-scrum-board#18]
## **Data & AI team**
- env : tensorflow 2

### Data Load & Pre-processing
> - 핀터레스트(Pinterest)에서 '힙한 패션', '클래식 패션' 등의 태그로 가 데이터 생성(fdata.zip) 5*10
> - 폴더 기준 동일 Label로  list형의 데이터 생성 및 numpy array 변환
> - 모델 입력을 위해 동일한 사이즈로 일괄 변경함(빠른 경과 확인을 위해 IMG_SIZE = 64)

### Modeling(CNN)
>- 이미지 분류에 좋은 성능을 보이는 기본적인 Convolutional Neural Network활용(keras layer이용)
>- Layers : Conv2D, Maxpool2D, Dropout, Dense, Flatten
>- activation : relu(일괄)
>- optimizer : 'adam'
>- model.h5로 전체 모델을 저장하고 그대로 불러와서 사용하는 방식
>>https://www.tensorflow.org/tutorials/keras/save_and_load

### Review
>- 수렴을 위해 더 많은 데이터 필요함. 또한 scoring이 목적이 아닌 실제 예측한 label을 태그로 활용할 것이기 때문에 validation loss <= 0.2, validation accuracy >= 0.95 를 목표로 모형 최적화 진행
>- 핀터레스트 파이썬 크롤링(하은)

**Comment by ywoo-park** [comment]
@Yeeun-Lee model 실행 return 결과 형태가 어떻게 되나요?

**Comment by Yeeun-Lee** [comment]
@ywoo21 model 실행 함수의 return결과의 경우 학습 기록을 담은 history가 반환되고, model 생성 함수의 경우 compile까지 완료된 모델이 반환됩니다!!

**Comment by ywoo-park** [comment]
@Yeeun-Lee 넵 알겠습니다!
